ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -378,7 +406,16 @@ func createEmptyCluster(context *cli.Context, ecsClient ecsclient.ECSClient, cfn
 		return fmt.Errorf("A CloudFormation stack already exists for the cluster '%s'.", commandConfig.Cluster)
 	}
 
-	if _, err := ecsClient.CreateCluster(commandConfig.Cluster); err != nil {
+	tags := make([]*ecs.Tag, 0)
+	var err error
```

Comment:
is this necessary? `err` is referenced in line 405 already no?

Reply:
Yeah, but 405 is an if statement, and so that `err` only has scope with in the if block.